internal/k8sinternal/runtime_test.go

```diff
@@ -80,25 +77,22 @@ func TestGetObjectMeta(t *testing.T) {
 	deployment := k8s.NewDeployment()
 	deployment.ObjectMeta = objectMeta
 	deployment.Spec.Template.ObjectMeta = podObjectMeta
-	assert.Equal(objectMeta, *k8s.GetObjectMeta(deployment))
-	assert.Equal(podObjectMeta, *k8s.GetPodObjectMeta(deployment))
+	assert.Equal(&objectMeta, k8s.GetObjectMeta(deployment))
```

Comment:
Out of curiosity: this looks functionally equivalent, is there an advantage to doing it this way?

Reply:
Gotcha, missed the type switch and thought it was some go convention I didn't know about 😄